test/util/service_context.go

```diff
@@ -0,0 +1,23 @@
+package util
```

Comment:
minor - there is already a file named `service_context.go` in the context package. Not a big deal since that is allowed by Go, but when searching for the file name sometimes can be confusing.

Reply:
Good catch, I didn't notice that. That was just a quick workaround for an import loop that I faced in unit tests. I believe that I'll drop this file soon as well. 
As for the name, any other name would have the same problem (service, context, etc...). Let's keep it and drop it as part of the next refactoring efforts

Reply:
I'm fine with keeping it and removing it later 👍